<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>173</width>
     <height>109</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="thickness">
        <property name="text">
         <string>Line thickness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="thickness_spinbox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="point_size">
        <property name="text">
         <string>Point size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="point_size_spinbox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="vertex_size">
        <property name="text">
         <string>Vertex size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vertex_size_spinbox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>158</width>
     <height>70</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="hist_range_label">
        <property name="text">
         <string>Hist Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="hist_range_spin">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="hist_nbins_label">
        <property name="text">
         <string>Hist NBins</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="hist_nbins_spin">
        <property name="minimum">
         <number>256</number>
        </property>
        <property name="maximum">
         <number>1024</number>
        </property>
        <property name="value">
         <number>512</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>22</y>
     <width>189</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="verbose_label">
        <property name="text">
         <string>Verbose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="verbose_spinbox">
        <property name="maximum">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="stepX_label">
        <property name="text">
         <string>StepX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="stepX_spin">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="stepW_label">
        <property name="text">
         <string>StepW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="stepW_spin">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="epsilon_label">
        <property name="text">
         <string>Epsilon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="epsilon_spin">
        <property name="decimals">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="frequency_label">
        <property name="text">
         <string>Update Freq</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="frequency_spin"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="max_iters_label">
        <property name="text">
         <string>Max Iters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="max_iters_spin"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="tau_label">
        <property name="text">
         <string>Tau</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="tau_spin">
        <property name="decimals">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
